src/components/hoc/Nautilus/README.md

```diff
@@ -1 +1,14 @@
-`Nautilus` is the base component (eg. the _higher-order component_ or _HOC_) used to inject the Nautilus Design System into your app. It provides a `theme` prop used to customise your Nautilus components, as well as injecting some sane default styles into your app. Your "root" components should be wrapped in the `<Nautilus>` component; any of your components that use a Nautilus component will need to be wrapped in a `<Nautilus>` component.
+`Nautilus` is the base component (eg. the _higher-order component_ or _HOC_) used to inject the Nautilus Design System into your app. It accepts a `theme` prop used to customise your Nautilus components, as well as injecting some sane default styles into your app. Your "root" components should be wrapped in the `<Nautilus>` component; any of your components that use a Nautilus component will need to be wrapped in a `<Nautilus>` component.
+
+`Nautilus` also accepts a `config` prop that can be used to customise the behaviour of Nautilus. For instance, if you want to tweak the default `Link` component you should supply a config to Nautilus:
```

Comment:
We'll probably want to detail all the different config options here eventually. For now it might be sufficient to say this is the only config value for now!

Reply:
I have some copy comments on this README in general but let's save that for a different PR. 😺